src/ui/src/builder/sidebar/BuilderSidebarToolkit.vue

```diff
@@ -32,12 +66,30 @@
 						:loader-max-height-px="18"
 					/>
 					<div class="name">{{ tool.name }}</div>
+					<button
+						v-if="
+							categoryId === 'Shared Blueprints' &&
+							isSharedBlueprintsEnabled &&
+							tool.sourceBlueprintId
+						"
+						class="tool__delete"
+						:data-writer-tooltip="`Delete ${tool.name}`"
+						data-writer-tooltip-placement="right"
+						@click.stop="
```

Comment:
Why do you need `stop` ?
```suggestion
						@click="
```